components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -546,5 +565,20 @@ private boolean isAuthPolicyAccountExistCheck() {
         return Boolean.parseBoolean(IdentityUtil.getProperty("AuthenticationPolicy.CheckAccountExist"));
     }
 
+    private boolean isAccountLockBypassRoleExist(UserStoreManager userStoreManager, String userName) throws AccountLockException {
```

Comment:
Shall we change the method name here to something like "isAccountLockByPassForUser" . **isAccountLockBypassRoleExist** gives the idea that this method checks whether a particular role exists. 
Rather this method, check whether a  particular role is assigned to the  user to determine, whether user is allowed to byPass account locking.